<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test627</class>
 <widget class="QWidget" name="test627">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test627</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>321</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Main Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose your movies directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="browseDirectories">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>260</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse Directories</string>
       </property>
      </widget>
      <widget class="QPushButton" name="toSearchPage">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>510</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Proceed to Search Page</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="status">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="createCache">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Create a Local Cache</string>
       </property>
      </widget>
      <widget class="QPushButton" name="connectCache">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect to Local Cache</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Status:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="importMovies">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>260</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Import Movies</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dirPath">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>210</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="search">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Keyword </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="keyword">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>121</width>
         <height>103</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="releaseYear">
          <property name="text">
           <string>Release Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="runtime">
          <property name="text">
           <string>Runtime</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QRadioButton" name="avgScore">
          <property name="text">
           <string>Average Score</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="title">
        <string>Sort By</string>
       </property>
       <widget class="QRadioButton" name="ASC">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Ascending</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="DSC">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>158</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Descending</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Search for Movies</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mainMenuButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Go Back to Main Menu</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>591</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>29</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>OR</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QTableWidget" name="scoresTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>561</width>
         <height>71</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>Scores</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>IMDB </string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rotten Tomatoes</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Metascore</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Average Score</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="goBack">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Go back</string>
       </property>
      </widget>
      <widget class="QWidget" name="movieInfo" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>771</width>
         <height>38</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="runtime_2">
          <property name="text">
           <string>runtime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="year">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Year&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileLoc">
          <property name="text">
           <string>File loc</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>771</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>749</width>
         <height>39</height>
        </rect>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
